src/components/projects/components/data.js

```diff
@@ -0,0 +1,288 @@
+export const tableData = [
+    {
+      key: '1',
```

Comment:
you can use chatgtp to create this data. or you can create unique key by your self